code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -30,6 +30,7 @@
 	var/eye_color = "#000000"
 
 	var/skin_tone = "caucasian1" //Skin tone
+	var/override_skin_tone = "" //Overrides normal skin_tone. Used for custom skin color. Uses Hex color.
```

Comment:
use the autodoc format for this so that mousing over this var in vsc shows the documentation